CONTRIBUTING.md

```diff
@@ -22,12 +22,11 @@
 5. Create a PR for the release branch
 6. Once the release PR is landed to the `main` branch, checkout the `main` branch locally (remember to pull the remote changes) and run `npm run build`
 7. Run `git tag <VERSION_NUMBER>` with the new version and push the tag to git
-8. Ensure that the infrastructure repository is in the same directory as `ably-js` and is up to date with the remote main branch
-9. Run `npm run grunt -- release:deploy` (requires the SDKDeveloper AWS role) - pushes commit and tag, then publishes to the Ably CDN
-10. Run `npm publish .` (should require OTP) - publishes to NPM
-11. Visit https://github.com/ably/ably-js/tags and add release notes to the release (generally you can just copy the notes you added to the CHANGELOG)
-12. For nontrivial releases: update the ably-js submodule ref in the realtime repo
-13. Update the [Ably Changelog](https://changelog.ably.com/) (via [headwayapp](https://headwayapp.co/)) with these changes (again, you can just copy the notes you added to the CHANGELOG)
+8. Run `npm publish .` (should require OTP) - publishes to NPM
+9. Visit https://github.com/ably/ably-js/tags and add release notes to the release (generally you can just copy the notes you added to the CHANGELOG)
+10. Run the GitHub action "Publish to CDN" with the latest version number
```

Comment:
Let's swap 9 and 10 around - we don't want to prematurely announce a release until we know the publish workflow has worked correctly.

Reply:
Sure, although this is only temporary anyway until we move the npm publishing into the CI too so I don't want to dwell on the wording too much

Reply:
Yeah that's fair, but I would have thought that particular step would stay even after the NPM CD stuff is done